TestAppTypescript/android/build.gradle

```diff
@@ -14,7 +14,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.3.1'
-        classpath 'com.google.gms:google-services:4.0.1'
+        classpath 'com.google.gms:google-services:4.1.0'
```

Comment:
did you validate push token is sent?

Reply:
+1 for testing Push every time we bump firebase dependencies.

Reply:
I've rolled back this change as it were firebase dependencies that were causing build issues with TestAppTypescript.